### PR TITLE
ppc64le platform support

### PR DESCRIPTION
**Pull request by mkumatag**
etcdserver : ppc64le platform support

[contributing]: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#contribution-flow

### CODE REVIEW

**Comment by heyitsanthony** [comment]
lgtm. Thanks!

**Comment by mkumatag** [comment]
seems like gofmt is failing, will amend the commit with proper fix.

File: Documentation/op-guide/supported-platform.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The following table lists etcd support status for common architectures and opera
 | arm64        | Linux            | Experimental | @glevand         |
 | arm          | Linux            | Unstable     |                  |
 | 386          | Linux            | Unstable     |                  |
+| ppc64le      | Linux            | Stable       | etcd maintainers |
```
Comment (xiang90) [inline Documentation/op-guide/supported-platform.md]: @mkumatag 

Can you put your github handler there? It relies on the Jenkins IBM provides.

Comment (mkumatag) [inline Documentation/op-guide/supported-platform.md]: Sure..

Comment (mkumatag) [inline Documentation/op-guide/supported-platform.md]: @xiang90 done, can you please check the updated commit. -Thanks.

**Comment by xiang90** [comment]
@mkumatag You can add etcd maintainers back along with you if you want. But I can merge this as it is. Totally up to you.

**Comment by mkumatag** [comment]
@xiang90 Done

**Comment by xiang90** [comment]
Merging.

**Pull request merged**